scripts/reporting/np_release_script.py

```diff
@@ -85,7 +86,7 @@ def replace_binge_groups_month(x):
     return result
 
 def main(args=None):
-    final_df = pd.read_csv(os.path.join(directory, "demographics.csv"),
+    final_df = pd.read_csv(os.path.join(directory, "subset.csv"),
```

Comment:
Can you include some version info here? For example, add a timestamp to the filename?